cl_dll/tri.cpp

```diff
@@ -57,4 +203,28 @@ void CL_DLLEXPORT HUD_DrawTransparentTriangles( void )
 
 	if ( g_pParticleMan )
 		 g_pParticleMan->Update();
+
+	// Draw server-side triggers with TriAPI instead
+
+	glDisable(GL_TEXTURE_2D);
```

Comment:
Does this work in software mode?

Reply:
Haven't tested but probably not, so yeah I could check for `IEngineStudio.IsHardware()` at that point then

Reply:
Done

Reply:
Oh, in fact for software support, it possible to replace `glDisable(GL_TEXTURE_2D)` / `glEnable(GL_TEXTURE_2D)` hack with:

```cpp
	static bool init = true;
	static HSPRITE white_sprite = 0;

	if (init)
	{
		white_sprite = gEngfuncs.pfnSPR_Load("sprites/white.spr");
		init = false;
	}

	if (white_sprite == 0)
		return;

	if (gEngfuncs.pTriAPI->SpriteTexture(const_cast<model_s*>(gEngfuncs.GetSpritePointer(white_sprite)), 0))
	{
		// Custom code
	}
```